experimental/Schemes/Types.jl

```diff
@@ -673,12 +616,21 @@ identifications given by the glueings in the `default_covering`.
       return ideal(OO(U), one(OO(U)))
     end
     function production_func(F::AbsPreSheaf, U::PrincipalOpenSubset)
+      haskey(ID, U) && return ID[U]
       V = ambient_scheme(U)
       IV = F(V)::Ideal
       rho = OOX(V, U)
       IU = ideal(OO(U), rho.(gens(IV)))
       return IU
     end
+    function production_func(F::AbsPreSheaf, U::SimplifiedSpec)
+      haskey(ID, U) && return ID[U]
+      V = original(U)
```

Comment:
Original goes all the way up to the original covering of X, right? Then is is fine.

Reply:
No. It only goes up one step.

Reply:
So we call F(V) one step upstairs, which then calls production_func with the appropriate input and we recurse our way up?

Reply:
Yes.